rotkehlchen/assets/converters.py

```diff
@@ -925,7 +925,6 @@ def asset_from_gemini(symbol: str) -> Asset:
         raise UnsupportedAsset(symbol)
 
     name = GEMINI_TO_WORLD.get(symbol, symbol)
-
```

Comment:
Hehe already addressed this in the merge of bugfixes to develop. Funny you did not get a conflict.